packages/html-webpack-plugin/index.test.js

```diff
@@ -0,0 +1,88 @@
+const Plugin = require('./index');
+const AppLoader = require('@talend/react-components/lib/AppLoader/constant').default;
+
+const DATA = {
+	head: [
+		{
+			tagName: 'link',
+			closeTag: true,
+			attributes: { rel: 'stylesheet', type: 'text/css', href: 'app.css' },
+		},
+	],
+	body: [
+		{ tagName: 'script', closeTag: true, attributes: { type: 'text/javascript', href: 'app.js' } },
+	],
+};
+
+const OPTIONS = {
+	bodyBefore: [],
+};
+
+describe('@talend/html-webpack-plugin', () => {
+	let plugin;
+	let compiler;
+	let compilation;
+	let fnc;
+	let callback;
```

Comment:
Can you rename those 2 vars to something more explicit ?
`fnc` --> `pluginExecFn` ?
`callback` --> `pluginExecCallback` ?